docs/slide/index.html

```diff
@@ -0,0 +1,49 @@
+<!Doctype html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1 maximum-scale=1 user-scalable=no">
+  <meta name="description" content="Really simple css animation">
+  <meta name="keywords" content="CSS, Animation, Easy as hell">
+  <meta name="author" content="linxea">
+  <title>Spaceship 🚀</title>
+  <link rel="shortcut icon" type="image/x-icon" href="../favicon.ico">
+  <link href="https://fonts.googleapis.com/css?family=Monoton" rel="stylesheet">
+
+  <style>
+      *{
+          background-color: black;
+          color: white;
+          font-size: 56px;
+          font-family: 'Monoton', cursive;
+
+      }
+      
+    p {
+  animation-duration: 3s;
```

Comment:
let's indent it nicely over here. :) you should try [prettier](https://prettier.io/) which automatically indents the code properly.

p {
   animation-duration: 3s;
   animation-name: slidein;
}